octobot_trading/enums.py

```diff
@@ -407,6 +407,9 @@ class WebsocketFeeds(enum.Enum):
     MARK_PRICE = 'mark_price'
     LAST_PRICE = 'last_price'
     ORDERS = 'orders'
+    LEDGER = 'ledger'
```

Comment:
Is there a difference with PORTFOLIO ?

Reply:
in ccxt those are different feeds 
![image](https://user-images.githubusercontent.com/9078616/211171721-ad7e07ed-91b3-4bd3-961d-3ebccb79deb9.png)

Reply:
What's the difference?

Reply:
![image](https://user-images.githubusercontent.com/9078616/211171955-c443be32-0fed-44c1-a3a5-1924abdcb809.png)

Reply:
Ok so should we disable account trades history when having ledger?